algod/Dockerfile

```diff
@@ -0,0 +1,72 @@
+ARG GO_VERSION=1.17.5
+FROM golang:$GO_VERSION-bullseye as builder
+
+ARG CHANNEL=nightly
+ARG URL=
+ARG BRANCH=
+ARG SHA=
```

Comment:
I would advocate for sane defaults here, if possible

Reply:
It's been too long since I worked on this. I don't remember why, but I had some issue with using defaults.